Formula/b/bld.rb

```diff
@@ -0,0 +1,28 @@
+class Bld < Formula
+  desc "Add *bld* to your shell to interact with your Build.io applications"
+  homepage "https://build.io"
+  url "https://github.com/buildio/cli/archive/refs/tags/v0.1.2.tar.gz"
+  sha256 "dbb138dd368b5e5aa71d9d3ac4a1d2e4660f765c64fb59fc7699d4c6d245acc2"
+  license "AGPL-3.0-or-later"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "libssh2" => :build
+  depends_on "crystal"
+
+  def install
+    mkdir bin
+    system "shards", "build", "--production", "--release", "--no-debug"
+    system "strip", "./bin/bld"
+    bin.install "./bin/bld"
+    ohai "----> Login to Build.io to get started."
+    ohai "      bld login"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/bld --version").chomp
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.